analyzer/src/funTest/kotlin/managers/Extensions.kt

```diff
@@ -32,5 +34,6 @@ fun Any?.toYaml() = yamlMapper.writeValueAsString(this)!!
 fun PackageManager.resolveSingleProject(definitionFile: File): ProjectAnalyzerResult =
     resolveDependencies(listOf(definitionFile))[definitionFile].let { result ->
         result shouldNotBe null
-        result!!
+        result!! should haveSize(1)
```

Comment:
`result!! should haveSize(1)` is not needed as covered by `single()` and imo should be removed.

Reply:
So is `shoudlNotBe null` covered by `!!`, but I prefer failing matchers with proper messages that make the test yellow over arbitrary exceptions that make the test red.